src/main/webapp/zoom.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="UTF-8">
+  <title>Seniors Connect</title>
+  <link rel="stylesheet" href="styles/techstyle.css">
+  <script src="script.js"></script>
+  <script src="scripts/login-script.js" defer></script>
+</head>
+<body>
+  <div id="nav-placeholder"> </div>
+
+  <div class="content">
+    <div class="header">
+      <h1>Zoom</h1>
+      <p>Description of Zoom</p>
+      <a href="https://zoom.us/"></a>
+    </div>
+
+    <div id = "helpful-links">
+      <li><a href="https://support.zoom.us/hc/en-us/articles/206618765-Zoom-video-tutorials">Zoom Video Tutorials</a></li>
+      <li><a href="https://support.zoom.us/hc/en-us/articles/201362193-Joining-a-meeting">Joining a Meeting</a></li>
+      <li><a href="https://support.zoom.us/hc/en-us/articles/201362613-How-Do-I-Host-A-Video-Meeting-">Hosting a Meeting</a></li>
+    </div>
+
+    <div id = "helpful-videos">
```

Comment:
Styling: Same as above. Spaces around the equal sign.

Reply:
Fixed.